src/user_summary/constants.py

```diff
@@ -0,0 +1,16 @@
+# File contains constants for user_summary app
+
+# Mapping of time filter with their time equivalent
+TIME_FILTER_MAPPING = {'1 hour': 1 / 24, '6 hours': 6 / 24,
```

Comment:
I think there is little practical value in the shorter ranges. The aim is to provide a tool that can be used to gauge someone's wiki activity in the context of their job performance, or job application; compering their overall activity with their activity for the last half year is useful (maybe they used to be a very active editor but not anymore), looking at someone's activity over the last week is maybe somewhat useful for a weekly review of how they are doing their job (although less likely + for such a small time scale you can just look at their edits directly), I can't imagine the data for the last hour being any good.

Reply:
Okay, I'll keep the minimum filter for 7 days.